### ISSUE TITLE
Pressing Start on Valkyrie 4.2.1 now shows a "Well that didn't work" error

### ISSUE THREAD

**Issue by samalex91** [RePod/psdle#119]
**Browser:** Edge 128

**Method:** Extension

**Online store version and language:** Desktop/en-us (en-au Store)
**PSDLE version:** 4.2.1 Valkyrie

---

**Problem:**
When pressing start on the transaction history page, it no longer prompts for a password and works. I receive an error stating
"Well, that didn't work
Wait a minute and try again."

**How to replicate:**
Go to PS Store transaction history page. PSDLE will show on top of the screen as per normal.
Press Start to see error.

**Comment by lance8765** [comment]
Yep, same here, Desktop/it-it in my case. I think it's been a week at this point.

**Comment by RePod** [comment]
At a glance it seems [CORS related](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) based on their server config. Basically, either intentionally or by accident, the server providing the Entitlements through the older API is not allowing access to the page(s) PSDLE is running on. It may or may not remedy itself in time.

There's nothing I can really do. Add-ons/extensions that bypass CORS may help but I have none to recommend.
~~As always when there's a new issue I have trouble signing in.~~

**Comment by lance8765** [comment]
> At a glance it seems [CORS related](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) based on their server config. Basically, either intentionally or by accident, the server providing the Entitlements through the older API is not allowing access to the page(s) PSDLE is running on. It may or may not remedy itself in time.
> 
> There's nothing I can really do. Add-ons/extensions that bypass CORS may help but I have none to recommend. ~As always when there's a new issue I have trouble signing in.~

Do you remember if this already happened in the past?

Also login issues seem to be quite common nowadays. I bet you're getting errors after inserting the password?

**Comment by samalex91** [comment]
Not experiencing any login issues personally. Usually pressing Start takes me to a login page where I have to input my password.

Now however, pressing Start posts the error before getting redirected to the password screen.

**Comment by lance8765** [comment]
I've never ever had to login after trying to start PSDLE.

However, it reloads the page once before i could finally generate the list.

**Comment by ccfman2004** [comment]
Same error me on FireFox.

**Comment by Piett1313** [comment]
I just tried to get a new listing of my PS3 download history today, and get this error on both Firefox and Chrome

**Comment by RePod** [comment]
Might be time to call it.

**Comment by RePod** [comment]
4.2.2 released which removes the Transaction History button from the extension popup.
The extension will still execute PSDLE Valkyrie as before in the hopes that it comes back, otherwise there's been a X/Close button in the top right corner for a while now so the panel can be used normally.

**Comment by induna-crewneck** [comment]
same issue with 4.2.2

`
psdle.valkyrie.js:2115 PSDLE | Ready.

The following strings need to be removed from app/pending-strings.js:
- vt.page.boku.reverify
- vt.request.code
- vt.please.wait
- vt.paypal.doesnt.open.automatically
- vt.button.continue
- vt.card.network.title

checkPendingCollisions @ ?nuDataSessionId=bc2dc1d0-dd8c-11ef-a866-a19fe8f2f389...

Unhandled error in promise: TypeError: Cannot read property 'xxx' of undefined.
tryCatcher @ ?nuDataSessionId=bc2dc1d0-dd8c-11ef-a866-a19fe8f2f389...

Fetch failed for resource: localization/en.json
fetchI18nResource @ ?nuDataSessionId=bc2dc1d0-dd8c-11ef-a866-a19fe8f2f389...

XHR request failed with status 403 (Forbidden).
xhr.onload @ ?nuDataSessionId=bc2dc1d0-dd8c-11ef-a866-a19fe8f2f389...

Promise rejection: Unexpected token < in JSON at position 0.
fileReaderReady @ ?nuDataSessionId=bc2dc1d0-dd8c-11ef-a866-a19fe8f2f389...
`

**Comment by RePod** [comment]
My [above comment](https://github.com/RePod/psdle/issues/119#issuecomment-2411564963) is still relevant, including the ability to dismiss it.

It was left in with the hopes it will come back although that is increasingly unlikely by the day.
If you do not need the barebones PS4/5 Library export you can stop using the extension.

**Comment by induna-crewneck** [comment]
What exactly do you mean by barebone library export?

**Comment by RePod** [comment]
Whatever is available from the [Library](https://library.playstation.com) (Gotham) PSDLE, which is not much.
After the initial push to support it I've not gone back to it.

**Comment by NiQ1** [comment]
> At a glance it seems [CORS related](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) based on their server config. Basically, either intentionally or by accident, the server providing the Entitlements through the older API is not allowing access to the page(s) PSDLE is running on. It may or may not remedy itself in time.
> 
> There's nothing I can really do. Add-ons/extensions that bypass CORS may help but I have none to recommend. ~As always when there's a new issue I have trouble signing in.~

Doesn't seem like CORS. I've tried with "--disable-web-security" and there was no difference.

**Comment by SlipSlot** [comment]
Okay cool so this is dead. Glad I wasted hours of my time trying to contribute.
